netconf/rpc.go

```diff
@@ -54,7 +55,7 @@ type RPCError struct {
 }
 
 func (re *RPCError) Error() string {
-	return fmt.Sprintf("netconf rpc [%s] '%s'", re.Severity, re.Message)
+	return fmt.Sprintf("netconf rpc [%s]: %s", re.Severity, strings.Trim(re.Message, "[\r\n]"))
```

Comment:
Can you please show me some cases where you need to remove leading and trailing "[\r\n]"?